whelk-core/src/main/groovy/whelk/component/PostgreSQLComponent.groovy

```diff
@@ -283,14 +358,18 @@ class PostgreSQLComponent implements Storage {
         
         GET_SYSTEMID_BY_IRI = "SELECT lddb__identifiers.id, lddb.deleted FROM lddb__identifiers "+
                 "JOIN lddb ON lddb__identifiers.id = lddb.id " +
-                "WHERE lddb__identifiers.iri = ? ";
+                "WHERE lddb__identifiers.iri = ? "
 
         GET_THING_MAIN_IRI_BY_SYSTEMID = "SELECT iri FROM $idTableName WHERE graphindex = 1 and mainid is true and id = ?"
         GET_DOCUMENT_BY_IRI = "SELECT lddb.id,lddb.data,lddb.created,lddb.modified,lddb.deleted FROM lddb INNER JOIN lddb__identifiers ON lddb.id = lddb__identifiers.id WHERE lddb__identifiers.iri = ?"
 
         GET_LEGACY_PROFILE = "SELECT profile FROM $profilesTableName WHERE library_id = ?"
     }
 
+    void logStats() {
+        // NOP
```

Comment:
What where you planning to put here?

Reply:
It's overridden in CachinPostgreSQLComponent so that the reindex job can log cache statistics after it is done.